backends/qualcomm/utils/utils.py

```
@@ -989,3 +995,135 @@ def tag_quant_io(gm: torch.fx.GraphModule, get_quant_io_dtype_fn: Callable):
+        self.filename = filename
+        self.directory = directory
+        self.py_op_wrapper_list = py_op_wrapper_list
+        self.dot = Digraph(filename, format="svg")
+        self.dot.attr(rankdir="TB")
+        self.dot_string = dot_string
+        self.draw()
+
+    def dfs_add_edges(self, node_name, visited, node_list):
+        if node_name in visited:
+            return
+        visited.add(node_name)
+
+        input_list = node_list[node_name]["input_list"]
+        for input_node_name in input_list:
+            self.dot.edge(input_node_name, node_name)
+            self.dfs_add_edges(input_node_name, visited, node_list)
+
+    def get_dot_graph(self):
+        visited = set()
+        node_list = {}
+
+        # get node
+        for py_op_wrapper in self.py_op_wrapper_list:
+            op_wrapper = py_op_wrapper.GetOpWrapper()
+            # TODO: multi output
+            for i in range(op_wrapper.GetOpConfig()["numOfOutputs"]):
+                if(op_wrapper.GetOpConfig()["outputTensors"][0].version == 1):
+                    node = op_wrapper.GetOpConfig()["outputTensors"][i].v1
+                    node_name = node.name
+                    input_list = []
+                    for j in range(op_wrapper.GetOpConfig()["numOfInputs"]):
+                            if(op_wrapper.GetOpConfig()["inputTensors"][j].version == 1):
+                                input_node = op_wrapper.GetOpConfig()["inputTensors"][j].v1
+                                input_node_name = input_node.name
+                                if input_node_name not in node_list:
+                                    node_list[input_node_name] = {"node" : input_node, "input_list" : []}
+                                input_list.append(input_node_name)
+                                # ToDo: tensor v2
+                            elif (op_wrapper.GetOpConfig()["outputTensors"][j].version == 2):
+                                pass
```

Could we raise an exception here to prevent drawing the wrong graph?